src/refinement-list/__tests__/refinement-list.spec.ts

```diff
@@ -289,7 +289,33 @@ describe('RefinementList', () => {
     const fixture = render({ items: [] });
     fixture.componentInstance.testedWidget.autoHideContainer = true;
     fixture.detectChanges();
+    expect(
+      fixture.debugElement.nativeElement.querySelector(
+        'ais-refinement-list > .ais-RefinementList'
+      )
+    ).toBeFalsy();
+    expect(fixture).toMatchSnapshot();
```

Comment:
I think we're fine without the snapshot tests since you've already added a meaningful assert above. what do you think?

Reply:
I thought they were better to look if the class is there and where it landed.

Reply:
makes sense 👍

Reply:
We don't need this noise IMO, we likely don't event read all these snapshots. This isn't even doing when the test name states.

We could just test that `element.classList` doesn't/does contain the `noRefinement` CSS class.